angular/core/components/thread_page/context_panel/context_panel.scss

```diff
@@ -33,6 +33,20 @@
   margin-top: 16px;
 }
 
-.context-panel__edit-link {
+.context-panel__edit,
+.context-panel__add-comment {
+  @include lmoBtnLink;
+  @include fontHelveticaMedium;
+  @include fontSmall;
   cursor: pointer;
+  text-transform: uppercase;
+  letter-spacing: .3px;
+  margin-right: 30px;
+}
+.context-panel__add-comment {
+  color: #5cb85c;
```

Comment:
It's not a new colour, it's the colour from the start proposal button etc, but I couldn't find a variable for it in the app